app/src/main/java/com/vanced/manager/core/ext/FragmentExt.kt

```diff
@@ -0,0 +1,15 @@
+package com.vanced.manager.core.ext
+
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import kotlin.reflect.full.createInstance
+
+fun Fragment.requireSupportFM() = requireActivity().supportFragmentManager
+
+inline fun <reified D : DialogFragment> Fragment.showDialogRefl() {
```

Comment:
And check this. Maybe we don't like reflect.

Reply:
showDialog() must be enough, I don't see a reason for keeping showDialogRef() as showDialog() does exactly the same